packages/babel-plugin-transform-react-jsx/src/create-plugin.ts

```diff
@@ -167,29 +170,25 @@ You can set \`throwIfNamespace: false\` to bypass this warning.`,
 
             if (file.ast.comments) {
               for (const comment of file.ast.comments) {
-                const sourceMatches = JSX_SOURCE_ANNOTATION_REGEX.exec(
-                  comment.value,
-                );
+                const value = `/*${comment.value}*/`;
```

Comment:
The `comment.value` is either a block comment without the wrapping `/* */` or an inline comment without the leading `//`. I think we don't need to add `/* */` back, instead we can remove `/* */` in the regex.

Reply:
I tried removing `/* */` in regex but it isn't working right it works same as old.

Reply:
I don't think I can follow. Aren't these propositions logically equivalent to each other?

```
s = " @jsx jsx " implies that /^\*?\s*@jsx\s+([^\s]+)\s*$/.test(s) is true
```

```
s = " @jsx jsx " implies that /^\/\*\*?\s*@jsx\s+([^\s]+)\s*\*\/$/.test(`/*${s}*/`) is true
```

Reply:
I think this proposition is equal and the first one will work-out.

Reply:
I will make a new commit with the change